packages/react-cosmos-config/src/index.js

```diff
@@ -1,11 +1,12 @@
 import path from 'path';
+import slash from 'slash';
 import resolveFrom from 'resolve-from';
 import importModule from 'react-cosmos-utils/lib/import-module';
 
 const resolveUserPath = (userPath, rootPath) =>
-  path.isAbsolute(userPath) ? userPath : (
+  slash(path.isAbsolute(userPath) ? userPath : (
```

Comment:
Yeah I see what you mean, `path.join` is called with `userPath` which could have any combination of forward/back slashes. So `slash` is indeed required. 👍